engine/runtime-instrument-common/src/main/scala/org/enso/interpreter/instrument/job/ProgramExecutionSupport.scala

```diff
@@ -443,7 +447,32 @@ object ProgramExecutionSupport {
     value: ExpressionValue
   )(implicit ctx: RuntimeContext): Unit = {
     val expressionId = value.getExpressionId
-    val methodCall   = toMethodCall(value)
+    if (value.isProgressUpdate()) {
+      val p = Api.ExpressionUpdate.Payload.Pending(
```

Comment:
What does `p` stand for, payload or pending or pending payload? Maybe `pp` is a better name?

Reply:
I see little difference between `p` and `pp`, but you are questioning the right part of code, @4e6!

How do you want to deliver the progress values and progress messages (`LOG {}:{}` message isn't yet being delivered) from the `ExecutionProgressObserver` to the IDE? 

I find the current `value.isProgressUpdate()` pretty ugly. Don't you want to redesign it somehow?

Reply:
After discussion with @4e6 we decided to keep the current state mostly intact: f6156c5d80